Dockerfile

```diff
@@ -2,24 +2,38 @@ FROM public.ecr.aws/docker/library/node:24.8.0-alpine3.21 AS base
 
 ARG CURL_VERSION=8.14.1-r2
 
+SHELL ["/bin/ash", "-o", "pipefail", "-c"]
+
 RUN apk add --no-cache \
-    python3=3.12.12-r0 \
-    make=4.4.1-r2 \
+    bash=5.2.37-r0 \
+    chromium=136.0.7103.113-r0 \
```

Comment:
Why do we need chromium for the main Dockerfile?

We are using another Dockerfile for Playwright and other tests where Chromium is involved

We should leave the main Dockerfile as thin as possible to improve build time for production and other CI checks that don't depend on the browser (unit tests and so on)

Please try to avoid adding such heavy dependencies to the main dockerfile